_build/maintainer_guide.rst

```diff
@@ -0,0 +1,243 @@
+Symfony Docs Maintainer Guide
+=============================
+
+The `symfony/symfony-docs`_ repository stores the Symfony project documentation
+and it's managed by the `Symfony Docs team`_. This article explains in detail
+some of those management tasks, so it's only useful for maintainers and not
+regular readers or Symfony developers.
+
+Merging Pull Requests
+---------------------
+
+Technical Requirements
+~~~~~~~~~~~~~~~~~~~~~~
+
+* `Git`_ installed and properly configured.
+* ``gh`` tool installed. This is a proprietary CLI tool which only Symfony team
+  members have access to.
+* Some previous Git experience, specially merging pull requests.
+
+First Setup
+~~~~~~~~~~~
+
+First, fork the <https://github.com/symfony/symfony-docs> using the GitHub web
+interface. Then:
+
+.. code-block:: terminal
+
+    # Clone your fork
+    $ git clone https://github.com/<YOUR-NAME>/symfony-docs.git
+
+    $ cd symfony-docs/
+
+    # Add the original repo as 'upstream' remote
+    $ git remote add upstream https://github.com/symfony/symfony-docs
+
+    # Add the original repo as 'gh' remote (needed for the 'gh' tool)
+    $ git remote add gh https://github.com/symfony/symfony-docs
```

Comment:
If you use `origin` as the main repository and `wouterj` (username) as your fork repository, the GH tool works without gh remote.

Reply:
This could be added as a note

Reply:
This is explained when installing `gh` ... so I prefer to not duplicate that here.